### PR TITLE
Add use cases to cameraX createCameraProvider

### PR DESCRIPTION
**Pull request by KasemJaffer**
This allows you to take a high quality picture without interrupting the video stream.

```kotlin

val imageCapture = ImageCapture.Builder()
      .setCaptureMode(ImageCapture.CAPTURE_MODE_MAXIMIZE_QUALITY)
      .build()

init {
  // Pass in imageCapture use case when creating the camera provider
  CameraXHelper.createCameraProvider(lifecycleOwner, arrayOf(imageCapture).let {
      if (it.isSupported(application)) {
          CameraCapturerUtils.registerCameraProvider(it)
  
          // Save cameraProvider for unregistration later.
          cameraProvider = it
      }
  }
}

fun takeHighQualityPicture() {
    val outputOptions = ImageCapture.OutputFileOptions.Builder(
        File(getApplication<Application>().filesDir, "high_quality_picture.jpg")
    ).build()

    imageCapture.takePicture(
        outputOptions,
        ContextCompat.getMainExecutor(getApplication()),
        object : ImageCapture.OnImageSavedCallback {
            override fun onImageSaved(outputFileResults: ImageCapture.OutputFileResults) {
                Log.d(TAG, "Image saved successfully: ${outputFileResults.savedUri}")
            }

            override fun onError(exception: ImageCaptureException) {
                Log.e(TAG, "Error capturing image", exception)
            }
        }
    )
}
```

### CODE REVIEW

File: livekit-android-camerax/README.md [hunk 1]
```diff
@@ -19,7 +19,10 @@ See our [release page](https://github.com/livekit/client-sdk-android/releases) f
 ### Register the CameraProvider
 
 ```
-CameraXHelper.createCameraProvider(lifecycleOwner).let {
```
Comment (davidliu) [inline livekit-android-camerax/README.md:22]: This one's unneeded since you add it specifically for the image capture section below.

**Comment by davidliu** [comment]
Please run `./gradlew spotlessApply` to fix any style errors. Also, will need you to sign the CLA in order to merge this into the repo.

**Comment by kasemj** [comment]
> Please run `./gradlew spotlessApply` from the project root to fix any style errors. Also, will need you to sign the CLA in order to merge this into the repo.

Thanks, I signed the CLA but the problem is that my commits are saying they're unverified. I'm working on getting that fixed and push shortly.

**Comment by KasemJaffer** [comment]
> Please run `./gradlew spotlessApply` from the project root to fix any style errors. Also, will need you to sign the CLA in order to merge this into the repo.

Thanks, I signed the CLA but the problem is that my commits are saying unverified. I'm working to fix it and push shortly.

**Comment by davidliu** [comment]
Hey, I think the issue is that the emails for your older commits are still in the wrong format, for example

```
commit ba37382db1c9fe0189d9b6f388ace2f490c13759
Author: kasem <“xxx.xxxx@email.com”>
Date:   Tue Nov 12 12:13:00 2024 -0800

    Update readme

```

The quotes within the `<>` are probably what's messing up the detection (should be just `<xxx.xxxx@email.com>` without the quotes).

There's a quick guide on how to amend the author field [here](https://www.git-tower.com/learn/git/faq/change-author-name-email), under the interactive rebase section. I highly suggest creating a separate branch to save a copy of the work before doing an interactive rebase, especially if you're not familiar with it since it can drop commits entirely. You can also just do it in another branch and open up a separate PR with the fixed commits.

**Comment by KasemJaffer** [comment]
> Hey @KasemJaffer, I think the issue is that the emails for your older commits are still in the wrong format, for example
> 
> ```
> commit ba37382db1c9fe0189d9b6f388ace2f490c13759
> Author: kasem <“xxx.xxxx@email.com”>
> Date:   Tue Nov 12 12:13:00 2024 -0800
> 
>     Update readme
> ```
> 
> The quotes within the `<>` are probably what's messing up the detection (should be just `<xxx.xxxx@email.com>` without the quotes).
> 
> There's a quick guide on how to amend the author field [here](https://www.git-tower.com/learn/git/faq/change-author-name-email), under the interactive rebase section. I highly suggest creating a separate branch to save a copy of the work before doing an interactive rebase, especially if you're not familiar with it since it can drop commits entirely. You can also just do it in another branch and open up a separate PR with the fixed commits.

You were right! thanks for the tip. All fixed up now.

**Review by davidliu — APPROVED** [review]

**Pull request merged**